src/TeaAsAService/Tea.Core/Domain/User.cs

```diff
@@ -72,5 +80,42 @@ public static User CreateLocalDevUser()
                 LastBrewTimeUtc = DateTime.UtcNow
             };
         }
+
+        static bool ValidatePassword(string password)
+        {
+            const int MIN_LENGTH = 8;
+
+            if (password == null) throw new ArgumentNullException();

```

Comment:
Check for is null or empty not just null and just return back false we don't need to throw an exception, so

If(string.isNullOrEmpty(password)) return false;